skfda/_utils/_utils.py

```diff
@@ -730,6 +730,21 @@ def _classifier_fit_depth_methods(
     return classes, class_depth_methods_
 
 
+def _fit_feature_transformer(
```

Comment:
Where is this function used?

Reply:
That function is used on issue 376, I forgot to delete it from this branch.